src/openrct2-ui/windows/Ride.cpp

```diff
@@ -2055,12 +2062,32 @@ static void window_ride_main_mouseup(rct_window* w, rct_widgetindex widgetIndex)
  */
 static void window_ride_main_resize(rct_window* w)
 {
-    const int32_t offset = gCheatsAllowArbitraryRideTypeChanges ? 15 : 0;
-    w->flags |= WF_RESIZABLE;
-    int32_t minHeight = 180 + offset;
+    int32_t minHeight = 180;
     if (theme_get_flags() & UITHEME_FLAG_USE_LIGHTS_RIDE)
-        minHeight = 200 + offset + RCT1_LIGHT_OFFSET
-            - (ride_type_has_flag(get_ride(w->number)->type, RIDE_TYPE_FLAG_NO_TEST_MODE) ? 14 : 0);
+    {
+        minHeight += 20 + RCT1_LIGHT_OFFSET;
+
+        auto ride = get_ride(w->number);
+        if (ride != nullptr)
+        {
+#ifdef __SIMULATE_IN_RIDE_WINDOW__
```

Comment:
I might have missed something, but why is this behind an ifdef?

Reply:
You can get four lights in the ride window with this compile-time flag.

Reply:
Why? That's asking for bitrot, and it's not even useful.

Reply:
I want to be sure that it won't be requested some time later, it took a lot of effort to add in the first place.

Reply:
If it gets requested, I will veto it anyway.

I'm ok with merging the PR with this intact (too important a feature to get bogged by this), but I do want to remove it at a later point.

Reply:
If we end up removing it in a separate commit, it can easily be reverted. I'm not too fussed either way.